### PR TITLE
Feature/benchmarking scripts

### PR DESCRIPTION
**Pull request by ffelten**
Relates to #12 

Wishlist:
- [x] Explicit seeds
- [x] Explicit wandb entity
- [x]  Shell script: algo-name, env-id, gamma, wandb entity, timesteps, seed
- [x] standardize algorithms init and train to take parameters above
- [x] [openrlbenchmark](https://github.com/openrlbenchmark/openrlbenchmark) stored run and extract plot

Example plot generated using this query on `openrlbenchmark` CLI:
```
 python -m openrlbenchmark.rlops \
    --filters '?we=openrlbenchmark&wpn=MORL-Baselines&ceik=env_id&cen=algo&metric=eval/eum' \
        'PCN' \
        'MultiPolicy MO Q-Learning' \
        'Envelope' \
    --env-ids deep-sea-treasure-v0 deep-sea-treasure-concave-v0 \
    --ncols 2 \
    --ncols-legend 1 \
    --output-filename compare.png \
    --report \
    --scan-history
```
image

### CODE REVIEW

**Comment by ffelten** [comment]
@LucasAlegre very much a WIP but I think it might look like this in the end if we stick to the default hyperparameters. What do you think?

File: morl_baselines/common/morl_algorithm.py [hunk 1]
```diff
@@ -205,25 +208,41 @@ def get_config(self) -> dict:
             dict: Config
         """
 
-    def setup_wandb(self, project_name: str, experiment_name: str) -> None:
+    def register_additional_config(self, ref_point: np.ndarray, known_front: Optional[List[np.ndarray]]) -> None:
```
Comment (LucasAlegre) [inline morl_baselines/common/morl_algorithm.py:211]: Could this method simply receive a dict instead? Then we could later be used to register anything we'd like.

File: morl_baselines/multi_policy/pareto_q_learning/pql.py [hunk 2]
```diff
@@ -216,19 +221,19 @@ def train(
...
-                pf = self._eval_all_policies()
-                log_all_multi_policy_metrics(
-                    current_front=pf,
-                    hv_ref_point=eval_ref_point,
-                    reward_dim=self.reward_dim,
-                    global_step=self.global_step,
-                    writer=self.writer,
-                    ref_front=known_pareto_front,
-                )
+            self.epsilon = max(self.final_epsilon, self.epsilon * self.epsilon_decay)
```
Comment (LucasAlegre) [inline morl_baselines/multi_policy/pareto_q_learning/pql.py:236]: I realized that we always use linearly decayed epsilon in the other algorithms, and here it uses exponentially decayed. Should we replace this by the linearly decayed so the exploration is the same?

**Review by LucasAlegre — CHANGES_REQUESTED** [review]

File: morl_baselines/common/morl_algorithm.py (hunk 1 above)
Comment (ffelten) [reply morl_baselines/common/morl_algorithm.py:211]: Yeah I can do that, I just liked the typing haha

File: morl_baselines/multi_policy/pareto_q_learning/pql.py (hunk 2 above)
Comment (ffelten) [reply morl_baselines/multi_policy/pareto_q_learning/pql.py:236]: It is exponentially decaying in the original paper though. I think we should keep it like this.

**Pull request merged**